app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -0,0 +1,414 @@
+<%
+  title ||= "GOV.UK - The best place to find government services and information"
+  html_lang ||= "en"
+%>
+
+<!DOCTYPE html>
```

Comment:
Will this whitespace get trimmed automatically? I believe [some older browsers will be forced into 'quirks mode' if there is whitespace before the doctype](https://github.com/alphagov/govuk-frontend/pull/949).